### ISSUE TITLE
[Steamshovel] Screenshot resets TextSummary position (Trac #1771)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1763]
Re-positioned text resets to default location when taking a screenshot. This causes TextSummaries to overlap.

Migrated from https://code.icecube.wisc.edu/ticket/1771
```json
{
    "status": "closed", 
    "changetime": "2016-07-22T15:41:28", 
    "description": "Re-positioned text resets to default location when taking a screenshot. This causes TextSummaries to overlap.", 
    "reporter": "wvandriessche", 
    "cc": "hdembinski", 
    "resolution": "fixed", 
    "_ts": "1469202088154509", 
    "component": "combo core", 
    "summary": "[Steamshovel] Screenshot resets TextSummary position", 
    "priority": "normal", 
    "keywords": "steamshovel textsummary screenshot", 
    "time": "2016-07-05T11:50:41", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160721T15:47:11: 
* olivas changed owner from "" to "hdembinski"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160722T15:41:28: 
* hdembinski commented:

> In changeset 148696:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148696"
> fixed TextSummary moving to default position bug, closes #1763
> ```


* hdembinski changed resolution from "" to "fixed"
* hdembinski changed status from "assigned" to "closed"
